articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -169,7 +154,7 @@ compile group: 'com.microsoft.azure', name: 'msal4j', version: '0.5.0-preview'
 
 ### MSAL initialization
 
-You can add the reference to MSAL4J by adding the following code to the top of the file where you will be using MSAL4J: 
+You can add the reference to MSAL4J by adding the following code to the top of the file where you will be using MSAL4J:
```

Comment:
More direct is: "Add a reference to the MSAL4J library by adding the following ..."

Reply:
Edited